Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,252 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background: #ffffff;
+  margin: 0 auto;
+  font-family: 'Open Sans', sans-serif;
 }
-
 /******************************************
 /* LAYOUT
 /*******************************************/
+nav {
+  float: right;
+  margin-right: 50px;
+  margin-top: 20px;
+}
+
+nav li {
+  margin-right: 20px;
+  display: inline;
+}
+
+header {
+  width: 100%;
+  background-color: #033048;
+}
+
+.logo {
+  margin-top: 0;
+  dislay: inline-block;
+}
+
 
 /******************************************
-/* PAGES
+/* SECTIONS
+/*******************************************/
+
+nav a {
+  text-decoration: none;
+  color: #f9e42e;
+  font-style: 700;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: 800;
+}
+
+header .logo {
+  color: #f0efef;
+  display: inline-block;
+  margin-left: 50px;
+  margin-top: 10px;
+
+}
+
+.content-wrapper{  
+  background: #f0efef;
+  margin: 0 auto;
+  width: 75%;
+  padding: 25px 50px 20px;
+
+}
+
+.articles,
+.sidebar {
+  float: left;
+}
+
+.articles {
+  width: 75%;
+  padding-left: 50px;
+}
+
+.article_images {
+  width: 100%;
+}
+
+.sidebar {
+  width: 25%;
+  background: #f0efef;
+  padding-left: 20px;
+}
+
+.160x600 {
+  padding-left: 20px;
+  padding-top: 60px;
+  padding-right: 50px;
+  float: left;
+}
+
+ul {
+  margin: 0;
+  list-style-type: none;
+}
+
+
+/******************************************
+/* PAGES --- I've actually come to the conclusion
```

Comment:
This is for when you have a site with multiple pages, and you would put individual page styles here, so like `/* ABOUT PAGE */` and then all the styles specific to the about page...etc